lib/send-mail.ts

```diff
@@ -60,15 +60,22 @@ export function parseMBox(mbox: string, gentle?: boolean):
     let subject: string | undefined;
     let to: string | undefined;
 
-    for (const line of header.split(/\n(?![ \t])/)) {
+    for (const line of header.split(/\n(?![ \t])/)
+        .map(e => e.replace("\r\n", " ").replace("\n", " "))) {
         const colon = line.indexOf(": ");
+
+        if (colon < 0 && line.lastIndexOf(":") === line.length - 1) {
+            // Keys without value are valid
+            continue;
+        }
         if (colon < 0) {
             throw new Error(`Failed to parse header line '${line}`);
         }
         const key = line.substr(0, colon);
-        const value = replaceAll(line.substr(colon + 2), "\n ", " ");
+        const value = replaceAll(line.substr(colon + 2), "\n ", " ").trim();
```

Comment:
Can the `trim` be removed from `case "from":`  below?

Reply:
Two different cases:

- This is for handling cases where the (whole) value part is starting on a second line.
- The below is about splitting the value part itself into multiple elements

Reply:
Thanks for clarifying.  Please see earlier comment.